src/Xamarin.Android.Build.Tasks/Utilities/LlvmIrGenerator/LlvmIrGenerator.cs

```diff
@@ -476,7 +477,12 @@ bool MaybeWritePreAllocatedBuffer<T> (StructureInfo<T> info, StructureMemberInfo
 
 			WriteGlobalSymbolStart (variableName, preAllocatedBufferVariableOptions, output);
 			ulong size = bufferSize * smi.BaseTypeSize;
-			output.WriteLine ($"[{bufferSize} x {irType}] zeroinitializer, align {GetAggregateAlignment ((int)smi.BaseTypeSize, size)}");
+			output.Write ('[');
+			output.Write (bufferSize);
+			output.Write (" x ");
+			output.Write (irType);
+			output.Write ("] zeroinitializer, align ");
+			output.WriteLine (GetAggregateAlignment ((int) smi.BaseTypeSize, size));
```

Comment:
For this line, for example, I could go through and add a comment like:
```csharp
// $"[{bufferSize} x {irType}] zeroinitializer, align {GetAggregateAlignment ((int)smi.BaseTypeSize, size)}"
```
I'll go through and do that, but let's see if the PR has any failing tests or anything.